test/blackbox-tests/test-cases/display.t

```diff
@@ -0,0 +1,153 @@
+Document how Dune displays various things
+=========================================
+
+  $ echo '(lang dune 3.0)' > dune-project
+  $ export BUILD_PATH_PREFIX_MAP=SH=`command -v sh`
+
+Errors with location embed in their output
+------------------------------------------
+
+  $ cat >dune<<"EOF"
+  > (rule
+  >  (alias default)
+  >  (action (system "echo 'File \"foo\", line 1: blah'; exit 42")))
+  > EOF
+
+  $ dune clean; dune build
+  File "foo", line 1: blah
+  [1]
+
+  $ dune clean; dune build --always-show-command-line
+            sh alias default (exit 42)
+  (cd _build/default && SH -c 'echo '\''File "foo", line 1: blah'\''; exit 42')
```

Comment:
I wonder why is `SH` capitalised?

Reply:
It's because of the `BUILD_PREFIX_MAP` at the beginning of the test that adds a rule to rewrite <path-to-sh> by `SH`. We do that to make the test more reproducible as the path to sh is not the same on all machine.

`BUILD_PREFIX_MAP` is a standard that's implemented by Dune's cram tests.

Reply:
Ah, got it, nice!